### ISSUE TITLE
outdated dataset_infos.json might fail verifications

### ISSUE THREAD

**Issue by songfeng** [huggingface/datasets#2038]
The [doc2dial/dataset_infos.json](https://github.com/huggingface/datasets/blob/master/datasets/doc2dial/dataset_infos.json) is outdated. It would fail data_loader when verifying download checksum etc..

Could you please update this file or point me how to update this file?

Thank you.

**Comment by lhoestq** [comment]
Hi ! Thanks for reporting.

To update the dataset_infos.json you can run:
```
datasets-cli test ./datasets/doc2dial --all_configs --save_infos --ignore_verifications
```

**Comment by lhoestq** [comment]
Fixed by #2041, thanks again @songfeng !
